test/monitoring/control-plane_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: Add AWS here once prometheus-operator is fully supported there
```

Comment:
This is not valid build tag.

BTW, we do support prometheus-operator on AWS, why can't we add it?

When using `examples/aws-testing` configuration, tests passes.

Reply:
We don't support prometheus on AWS. We only do unit tests there not e2e tests. And you won't find any prometheus config block in CI config of AWS:
https://github.com/kinvolk/lokomotive/blob/master/ci/aws/aws-cluster.lokocfg.envsubst

Reply:
Hmm, right. We could/should enable it, but on the other hand, that will raise the cost of the AWS pipeline :thinking: